### DISCUSSION TITLE
Github page stopped working! :(

### DISCUSSION THREAD

**Discussion by jasonchenscie** [community/community#21625] [Other Feature Feedback, Questions, & Ideas]
<p>Hello there!</p>
<p>I’ve been digging the web but haven’t found a solution. I have my github pages <a href="http://jasonchenscie.github.io" rel="nofollow noopener">jasonchenscie.github.io</a> up for about 2 years now I believe, but somehow the other day when I went on it, it only loads the title of the index.HTML but not anything after it… Any ideas why this is happening?</p>
<p>Thanks!</p>
<p>Jason</p>

**Comment by jasonchenscie** [comment]
<p>Bumping this a bit in case anyone has time to help me take a look!</p>

**Comment by AndreaGriffiths11** [comment]
<p>Hi <a class="mention" href="/jasonchenscie">@jasonchenscie</a>,</p>
<p>Thanks for being here! I took a look at you site and it seems to load okay on my end. All tabs and links worked properly what am I missing? </p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/2446ee33b2a5f7faa39b5a403598b6cac6f4b349.png" data-download-href="/uploads/short-url/5aV9IFgOycO4W05qhpcrYQMU9ER.png?dl=1" title="Screen Shot 2018-11-19 at 6.25.21 PM.png"><img src="https://user-images.githubusercontent.com/20666190/181065882-7d897b05-ab81-42cb-82f2-1209279458b3.png" alt="Screen Shot 2018-11-19 at 6.25.21 PM.png" data-base62-sha1="5aV9IFgOycO4W05qhpcrYQMU9ER" width="690" height="370" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/2446ee33b2a5f7faa39b5a403598b6cac6f4b349_2_690x370.png, https://aws1.discourse-cdn.com/github/original/2X/2/2446ee33b2a5f7faa39b5a403598b6cac6f4b349.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/2446ee33b2a5f7faa39b5a403598b6cac6f4b349.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/2446ee33b2a5f7faa39b5a403598b6cac6f4b349_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-11-19 at 6.25.21 PM.png</span><span class="informations">997×535 51 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jasonchenscie** [comment]
<p>Hello <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>!</p>
<p>That was weird, I wonder if something is wrong with my CNAME setup? This is what I see on my end </p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/4/4a84c29210a3b2a2e59a27e41c302d23b2142e4a.png" data-download-href="/uploads/short-url/aDdLJsj8jrEwJN8TTCZPui5Bi4y.png?dl=1" title="Screen Shot 2018-11-27 at 11.03.36 AM.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/32629964/181065884-466e0106-ba12-4f43-bc7f-ef6b94f4f5cf.png" alt="Screen Shot 2018-11-27 at 11.03.36 AM.png" data-base62-sha1="aDdLJsj8jrEwJN8TTCZPui5Bi4y" width="690" height="331" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/4/4a84c29210a3b2a2e59a27e41c302d23b2142e4a_2_690x331.png, https://aws1.discourse-cdn.com/github/optimized/2X/4/4a84c29210a3b2a2e59a27e41c302d23b2142e4a_2_1035x496.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/4/4a84c29210a3b2a2e59a27e41c302d23b2142e4a_2_1380x662.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/4/4a84c29210a3b2a2e59a27e41c302d23b2142e4a_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Screen Shot 2018-11-27 at 11.03.36 AM.png</span><span class="informations">2858×1372 122 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jasonchenscie** [comment, marked as answer]
<p>I figured it out! Thanks <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> for letting me know there is nothing wrong with my github repo, turns out it is an outdated IP address in the DNS setting. For those who run into the same problem, simply reconfigure your DNS per <a href="https://help.github.com/articles/troubleshooting-custom-domains/" rel="nofollow noopener">https://help.github.com/articles/troubleshooting-custom-domains/ </a></p>

**Comment by Crazz-Zaac** [comment]
<p>I’m also facing similar issue in my <a href="https://crazz-zaac.github.io/" rel="noopener nofollow ugc">githubpage</a>. I’m using <a href="https://jekyllthemes.io/theme/flexible-jekyll" rel="noopener nofollow ugc">flexile-jekyll theme</a>. The page was loading properly and everything seemed to work fine for about a month. Then later when I again checked, nothing seems to load properly. Whichever link I click, it just loads up the home page but without the effect of css and all. So messy. Help please!!</p>
